articles/quickstart/webapp/php/01-login.md

```diff
@@ -13,6 +13,13 @@ useCase: quickstart
 github:
   path: 00-Starter-Seed
 ---
+
+::: panel Beta Available
```

Comment:
Do we also want to encourage developers to raise issues on the GitHub repo or is community sufficient enough?

Was also about to suggest that this panel is pulled out into an include file but seeing as it's intended to be short lived, probably doesn't matter much.

Reply:
Hey Steve 👋 

Good call, links to the repo for bug reports are a good idea, I've added those.

Yeah, making the panel a reusable template and configurable for different QS links would be a good way to go, but probably more effort than it's worth for how short-lived this will be, I think 😄